src/components/Comments/Comments.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import {
+  addCommentOfGoal,
+  fetchGoalComments,
+  archiveCommentOfGoal,
+  updateGoalComment,
+} from '../../goal-comments/actions'
+import moment from 'moment'
+import './Comments.css'
+import Avatar from '../Avatar/Avatar'
+import Button from '../Button/Button'
+function Comment(props) {
+  return (
+    <div className='comment'>
+      {console.log(props.avatarUrl)}
+      <div className='avatar_comment_container'>
+        <Avatar avatar_url={props.avatarUrl} />{' '}
+      </div>
+      <div>
+        <div className='info'>
+          <span>{props.name}</span>
+          <span className='date'>
+            {moment
+              .unix(props.unix)
+              .tz('America/Caracas')
+              .format('Do,hh:mm a')}
+          </span>
+        </div>
+        <span>{props.content}</span>
+      </div>
+    </div>
+  )
+}
+function Comments(props) {
+  const inputRef = React.createRef()
```

Comment:
Instead of using `refs` let's use something a little more like... 
```javascript
import React, { useState } from "react";

export function NameForm(props) {
  const [name, setName] = useState("");
  
  const handleSubmit = (evt) => {
      evt.preventDefault();
      alert(`Submitting Name ${name}`)
  }
  return (
    <form onSubmit={handleSubmit}>
      <label>
        Frirst Name:
        <input
          type="text"
          value={name}
          onChange={e => setName(e.target.value)}
        />
      </label>
      <input type="submit" value="Submit" />
    </form>
  );
}
```
taken as an example from https://rangle.io/blog/simplifying-controlled-inputs-with-hooks/